### PR TITLE
Metrics Glossary: Precision

### PR DESCRIPTION
**Pull request by y27choi**
### Linked issue(s):
[KOL-2542](https://linear.app/kolena/issue/KOL-2542/port-accuracyprecisionrecall-doc-to-docskolenaio)

### What change does this PR introduce and why?
Added Precision metrics glossary doc

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

**Comment by gordonhart** [comment]
Let me know what you think @y27choi. I pared the content down a bit, particularly in the `Limitations and Biases` section.

File: docs/metrics/accuracy.md
Comment (y27choi) [inline docs/metrics/accuracy.md:1]: line 22 - should we change `predictions` to `inferences` here as well?

File: docs/metrics/precision.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+The precision metric is designed to measure the percentage of **positive predictions** that a model correctly predicted,
+ranging from 0 to 1 (where 1 is best).
+
+As shown in this diagram, precision is the fraction all inferences that are correct:
+
+$$\text{Precision} = \frac{\text{TP}}{\text{TP} + \text{FP}}$$
+
+Where $\text{TP}$ is the number of true positive inferences and $\text{FP}$ is the number of false positive inferences.
+
+In other words, precision measures the ability of a model to not label a negative sample as positive.
```
Comment (y27choi) [inline docs/metrics/precision.md]: Are we dropping this?

File: docs/metrics/precision.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+different ways to aggregate per-class precision scores: **macro**, **micro**, and **weighted**. Read more about these
+methods in the [Averaging Methods](./averaging-methods.md) guide.
+
+## Limitations and Biases
+
+As seen in its formula, precision only takes **positive** inferences (TP and FP) into account; negative inferences
+(TN and FN) are not considered. Thus, precision only provides one half of the picture, and should always be used in
+tandem with recall: recall penalizes false negatives, whereas precision does not.
+
+For a single metric that takes both precision and recall into account, use F1 score, which is the [harmonic] mean between precision
```
Comment (y27choi) [inline docs/metrics/precision.md]: Are these square brackets added on purpose?

**Review by gordonhart — APPROVED** [review]

**Comment by y27choi** [comment]
Finished reviewing.  Looks great 👍 — I like the conciseness of Limitations and Biases section.

**Pull request merged**